bucket/win-gpg-agent.json

```diff
@@ -34,6 +34,18 @@
         "    \"  # Use line below to enable SSH_AUTH_SOCK to point to cygwin socket`r`n\",",
         "    \"  #   openssh: cygwin`r`n\"",
         "  ) -join '' | Set-Content \"$dir\\agent-gui.conf\" -Encoding ASCII",
+        "}",
+        "if (!(Test-Path \"$dir\\pinentry.conf\")) {",
+        "  $content = \"gui:`r`n\"",
+        "  $content += \"  debug: false`r`n\"",
+        "  $content += \"  pin_dialog:`r`n\"",
+        "  $content += \"    delay: 500ms`r`n\"",
+        "  $content | Set-Content \"$dir\\pinentry.conf\" -Encoding ASCII",
+        "}",
+        "if (!(Test-Path \"$dir\\sorelay.conf\")) {",
```

Comment:
This may be better, but would not I want to have all 3 configuration files handled exactly the same way?

Reply:
Yes, this makes all three the same (test in $persist_dir, create in $dir)

Reply:
I was under impression that test is here to avoid overwriting configuration that user created by hand, making sure that user changes are persisted between versions. As such if I would test in $persist_dir and create in $dir I will either loose user changes or get an error (depending on how creating is implemented). Meantime test and creation in $dir has been there and functioning for many versions. @rashil2000 could you please elaborate a bit on the reasons behind proposed change?

Reply:
Not really sure what template file is, however, I am pretty sure that the way it is presently setup no links are created - simple small files are copied instead. Can we keep it the way it is? I see no benefits in what you are proposing. I do not really want to complicate matters and to create additional logic of testing in multiple places.